spec/features/dams_object_spec.rb

```diff
@@ -500,6 +500,30 @@
     expect(page).to have_selector('li', text: 'thale-cress')
     expect(page).to have_selector('li', text: 'thale-cress component')
   end
+
+  it 'should display component pager' do
+    visit dams_object_path(@damsComplexObj.pid)
+    expect(page).to have_selector('#component-pager')
+  end
+
+  it 'testing componenet pager functionality' do
```

Comment:
@dt-ucsd let's mark this test as [pending](https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples) for now so the test suite passes. i'm going to setup a ticket for the coming sprint to look into headless alternatives such as PhantomJS or Jasmine